File: Estrutura_Repetição/Python/Ex1101.py
while True:
    x, y = map(int, input().split())
    menor = min(x, y)
    maior = max(x, y)

    if menor <= 0 or maior <= 0:
        break
    else:
        soma = 0
        for i in range(menor, maior + 1):
            soma += i
            print(i, end= " ")
        print(f"Sum={soma}")

File: Estrutura_Repetição/JavaScript/Ex1101.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.on('line', (input) => {
    const [x, y] = input.split(' ').map(Number);

    if (x <= 0 || y <= 0) {
        rl.close();
    } else {
        let menor = Math.min(x, y);
        let maior = Math.max(x, y);
        let soma = 0;

        for (let i = menor; i < maior + 1; i++) {
            soma += i;
            process.stdout.write(i + ' ');
        }

        console.log('Sum=' + soma);
    }
});